spinta/backends/mongo.py

```diff
@@ -110,12 +110,12 @@ def push(context: Context, model: Model, backend: Mongo, data: dict, *, action:
     model_collection = backend.db[model.get_type_value()]
 
     # FIXME: hack to remove built-in properties
-    if data['id'] is None:
-        del data['id']
-    if data['revision'] is None:
-        del data['revision']
-    if data['type'] is None:
-        data['type'] = model.name
+    # if data['id'] is None:
+    #     del data['id']
+    # if data['revision'] is None:
+    #     del data['revision']
+    # if data['type'] is None:
+    #     data['type'] = model.name
```

Comment:
Why these are commented out?

Reply:
https://github.com/atviriduomenys/spinta/pull/28/commits/6f0e4d3c081be8700ba30e69dd569ce9dd32569b

> Hardcoded changes made to make qvarn-like revisions work, though it
basically broke every spinta's test. For now - roll back (or comment
out) these changes for the sake, that tests would be green.

Reply:
I've commented them out - as it's highly likely that they will be uncommented

Though following best practices it would've been best to just remove them overall..

Reply:
Could you give me an example, of how tests were broken?